.ci/matrix_job_release.yaml

```diff
@@ -20,14 +20,19 @@ steps:
   - name: Build Plugin
     agentSelector: "{nodeLabel: 'SWX-CI-DOCKER'}"
     run: |
+      if [ -e "/auto/mswg/release/ufm/plugins/${Plugin_name}/ufm-plugin-${Plugin_name}_${PLUGIN_VERSION}*" ];then 
+        exit 1
```

Comment:
please print the reason for the failure before you exit.
you want to see and not guess the reason of the failure